backend/src/main/resources/application.properties

```diff
@@ -24,7 +24,7 @@ mp.openapi.extensions.smallrye.operationIdStrategy=CLASS_METHOD
 
 quarkus.http.access-log.enabled=true
 quarkus.http.access-log.category=access_log
-quarkus.http.access-log.pattern=combined
+quarkus.http.access-log.pattern="%h %l %u %t \"%r\" %s %b orgId=%{d,x-rh-rbac-org-id}"
```

Comment:
The new pattern removes some information that we logged before.

Before:
```
2022-09-21T08:51:28.8959224Z 2022-09-21 08:51:28,889 INFO  [info] (executor-thread-0) 127.0.0.1 - user 21/Sep/2022:08:51:28 +0000 "GET /api/integrations/v1.0/endpoints/9de0632b-d4ab-4e16-9f29-af7ff1b7c87c/history HTTP/1.1" 200 241 "-" "Apache-HttpClient/4.5.13 (Java/11.0.11)"
```
After:
```
2022-09-21T09:03:18.0108475Z 2022-09-21 09:03:17,919 INFO  [info] (executor-thread-0) "127.0.0.1 - user 21/Sep/2022:09:03:17 +0000 "GET /api/integrations/v1.0/endpoints/82c774a5-abdd-4e3f-8461-9fa7c6d7efc1/history HTTP/1.1" 200 961 orgId=someOrdId"
```

Reply:
The pattern would need  `"%{i,Referer}" "%{i,User-Agent}"` at the end according to https://quarkus.io/guides/http-reference#configuring-http-access-logs

Reply:
Thanks, I'll push a new commit to try it.